roles/ipasmartcard_server/library/ipasmartcard_server_get_vars.py

```diff
@@ -116,21 +116,27 @@
 
 import sys
 from ansible.module_utils.basic import AnsibleModule
-from ipaplatform.paths import paths
+MODULE_IMPORT_ERROR = None
```

Comment:
I'd prefer that this variable is set after the fact, in the `else` clause for the `try`.

Reply:
Done